test/statetest/statetest_loader.cpp

```diff
@@ -128,14 +128,13 @@ state::AuthorizationList from_json<state::AuthorizationList>(const json::json& j
     for (const auto& a : j)
     {
         state::Authorization authorization{};
-        authorization.chain_id = from_json<intx::uint256>(a.at("chainId"));
+        authorization.chain_id = from_json<uint256>(a.at("chainId"));
         authorization.addr = from_json<address>(a.at("address"));
         authorization.nonce = from_json<uint64_t>(a.at("nonce"));
-        if (a.contains("signer"))
-            authorization.signer = from_json<address>(a.at("signer"));
-        authorization.r = from_json<intx::uint256>(a.at("r"));
-        authorization.s = from_json<intx::uint256>(a.at("s"));
-        authorization.v = from_json<intx::uint256>(a.at("v"));
+        authorization.signer = load_if_exists<address>(a, "signer");
```

Comment:
I think it sets it to 0 address, when not present in the test, but those are tests with invalid signature and it should stay `std::nullopt`

Reply:
Yes, correct. I noticed this by state tests. Fixed now.